docs/reference/manifest/appdomains.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: AppDomains element in the manifest file
-description: Lists any domains in addition to the domain specified in the `SourceLocation` element that your Office Add-in will use to load pages.
-ms.date: 07/03/2019
+description: Lists any domains in addition to the domain specified in the `SourceLocation` element that your Office Add-in will use and should be trusted by Office.
+ms.date: 06/12/2020
 localization_priority: Normal
 ---
 
 # AppDomains element
 
-Lists any domains in addition to the domain specified in the `SourceLocation` element that your Office Add-in will use to load pages. It also lists trusted domains from which Office.js API calls can be made from IFrames within the add-in. For each additional domain, specify an AppDomain element.
+Lists any domains, in addition to the domain specified in the `SourceLocation` element, that your Office Add-in will use and that should be trusted by Office. Among other things, this enables pages in the domains to make calls to Office.js APIs from IFrames within the add-in. For each additional domain, specify an **AppDomain** element.
```

Comment:
```suggestion
Lists any domains, in addition to the domain specified in the `SourceLocation` element, that your Office Add-in will use and that should be trusted by Office. This enables pages in the domains to make calls to Office.js APIs from IFrames within the add-in. For each additional domain, specify an **AppDomain** element.
```
I don't know if among will translate properly (could be that it will translate as physically among).

Reply:
I don't want to just remove it, because then it would imply that it only has this one effect. I'll change it to "This enables pages in the domains to make calls to Office.js APIs from IFrames within the add-in and has other effects."